src/Http/CakeHttpEventListener.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace CakeSentry\Http;
+
+use Cake\Event\EventListenerInterface;
+use Cake\Http\Client\ClientEvent;
+use Sentry\SentrySdk;
+use Sentry\Tracing\SpanContext;
+use Sentry\Tracing\SpanStatus;
+
+class CakeHttpEventListener implements EventListenerInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function implementedEvents(): array
+    {
+        return [
+            'HttpClient.afterSend' => 'handleAfterSend',
+        ];
+    }
+
+    /**
+     * @param \Cake\Http\Client\ClientEvent $event
+     * @return void
+     */
+    public function handleAfterSend(ClientEvent $event): void
+    {
+        $request = $event->getRequest();
+        $response = $event->getResult();
+
+        $parentSpan = SentrySdk::getCurrentHub()->getSpan();
+
+        if ($parentSpan !== null) {
+            $context = SpanContext::make()
+                ->setOp('http.client');
+            $span = $parentSpan->startChild($context);
```

Comment:
You need to start the span on `HttpClient.beforeSend` and end it on `HttpClient.afterSend`.